components/automate-cli/cmd/chef-automate/automateClusterCtlUtil.go

```diff
@@ -184,9 +184,21 @@ func doBootstrapEnv(airgapBundlePath string, saas bool) error {
 		return status.Annotate(err, status.DeployError)
 	}
 	if offlineMode {
-		err := moveFrontendBackendAirgapToTransferDir(airgapMetadata, airgapBundlePath)
-		if err != nil {
-			return status.Annotate(err, status.DeployError)
+		if frontend {
+			err := moveAirgapFrontendBundlesOnlyToTransferDir(airgapMetadata, upgradeRunCmdFlags.airgap)
```

Comment:
upgradeRunCmdFlags.airgap -> airgapBundlePath

Reply:
while testing i found that the value of upgradeRunCmdFlags.airgap is nil, which create the problem while fresh deployment for HA